src/main/java/org/ow2/proactive/catalog/report/TableDataBuilder.java

```diff
@@ -87,15 +95,35 @@ public void buildTableData(Set<CatalogObjectMetadata> orderedObjectsPerBucket, B
 
     }
 
+    private String getGenericInfo(CatalogObjectMetadata catalogObject) {
+        return catalogObject.getMetadataList()
+                            .stream()
+                            .filter(metadata -> metadata.getLabel().equals("generic_information"))
+                            .map(metadata -> "<li><b>" + metadata.getKey() + "</b> = " + metadata.getValue() + "</li>")
+                            .collect(Collectors.joining(""));
+
+    }
+
+    private String getVariable(CatalogObjectMetadata catalogObject) {
+        return catalogObject.getMetadataList()
+                            .stream()
+                            .filter(metadata -> metadata.getLabel().equals("variable"))
+                            .map(metadata -> "<li><b>" + metadata.getKey() + "</b> = " + metadata.getValue() + "</li>")
```

Comment:
If you are modifying the variable to include html code then rename the method to indicate it please. This is not the behavior that I would expect for such method name.